internal/controller/management_controller.go

```diff
@@ -211,14 +222,53 @@ func (r *ManagementReconciler) Update(ctx context.Context, management *kcm.Manag
 	return ctrl.Result{}, nil
 }
 
+// startDependentControllers starts controllers that cannot be started
+// at process startup because of some dependency like CRDs being present.
+func (r *ManagementReconciler) startDependentControllers(ctx context.Context, management *kcm.Management) (requue bool, err error) {
```

Comment:
this is effectively a reconcile, so like starting reconcile in reconcile, not against that, just mentioning how far kubebuilder is from reality

no action needed here